storage/src/mmr/mod.rs

```diff
@@ -1,43 +1,35 @@
 mod mem;
-pub use mem::{verify_proof, verify_range_proof, InMemoryMMR};
+pub use mem::{verify_proof, verify_range_proof, InMemoryMmr};
```

Comment:
nit: seems like stutter to have `InMemoryMmr` be in the `mem` crate? Suggest we consider just calling Mmr?

Reply:
I had a "pub use mem::{...InMemoryMmr} so you wouldn't generally use the crate name, but I could remove that and require it (or at least require you declare your own use).  You'd know better than I would what's more idiomatic in rust?

Reply:
This made me think though that I should move the "verify_" functions outside of mem because they are totally independent of how the MMR stores the data.

Reply:
I have moved the non-inmem-specific  code out of "mem" to the top level crate & renaming InMemoryMmr to "Mmr" within mem crate.